### DISCUSSION TITLE
Not able to install nillion in Windows

### DISCUSSION THREAD

**Discussion by aeyshubh** [NillionNetwork/discussions#31] [Q&A] [answered]
When i run curl https://nilup.nilogy.xyz/install.sh , I get : 
![image](https://github.com/NillionNetwork/discussions/assets/50445649/d6a67409-8cba-445b-823c-385c3c49dffb)

**Comment by Davetbutler** [comment, marked as answer]
Hey @aeyshubh. As stated in the [limitations here](https://docs.nillion.com/limitations#platforms) we do not yet support windows. If you are able to access a Mac or Linux machine/environment and have more questions, please just reach out to us :)

**Comment by jwolski2** [comment]
hey @aeyshubh. As @Davetbutler stated, we don't currently support Windows, but we would like to make this more apparent for Windows users that are following our docs and running the documented commands.

I see that you're running `curl https://nilup.nilogy.xyz/install.sh`, but you aren't piping the output to `bash` to perform the install:

```
curl https://nilup.nilogy.xyz/install.sh | bash
```

I don't expect the above to work for Windows users. But can you run the above and tell us the behavior you experience? It would help us improve the UX for Windows users.

**Comment by Davetbutler** [comment]
Hey @aeyshubh. We are now supported on Windows (through WSL). [You can see more details in our guide here](https://docs.nillion.com/installation#windows-guide).

Let me know how you get on, would love to see you submit to one of our builder bounties and win some prizes. [You can find them here](https://github.com/orgs/NillionNetwork/projects/3) 🎉
